### PR TITLE
CBnT/feature/cbfs support 2

### PR DESCRIPTION
**Pull request by ArthurHeymans**

### CODE REVIEW

**Review by ChriMarMe — DISMISSED** [review]

File: cmd/cbnt-prov/cmd.go [hunk 1]
```diff
@@ -416,11 +416,13 @@ func (g *generateBPMCmd) Run(ctx *context) error {
 			se.DigestList.List[iterator].HashAlg = g.IbbHash[iterator]
 		}
 
-		seg := *bootpolicy.NewIBBSegment()
-		seg.Base = g.IbbSegbase
-		seg.Size = g.IbbSegsize
-		seg.Flags = g.IbbSegFlag
-		se.IBBSegments = append(se.IBBSegments, seg)
+		if g.IbbSegbase != 0 {
```
Comment (ChriMarMe) [inline cmd/cbnt-prov/cmd.go:419]: I think this will break if you don't supply -iibsegbase=0

File: pkg/provisioning/cbnt/tools.go [hunk 2]
```diff
@@ -419,3 +421,42 @@ func StitchFITEntries(biosFilename string, acm, bpm, km []byte) error {
...
+// additional IBBSegment.
+func FindAdditionalIBBs(imagepath string) ([]bootpolicy.IBBSegment, error) {
+	ibbs := make([]bootpolicy.IBBSegment, 0)
+	image, err := os.Open(imagepath)
+	if err != nil {
+		return nil, err
+	}
+	defer image.Close()
+
+	imageI, err := image.Stat()
```
Comment (walterchris) [inline pkg/provisioning/cbnt/tools.go:435]: ImageI ?

File: cmd/cbnt-prov/cmd.go (hunk 1 above)
Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go:419]: I had the same issue several times in the past. If you check on the existence of an optional argument which wasn't supplied, it is not set at all and will result in a null pointer exception.

File: pkg/provisioning/cbnt/tools.go (hunk 2 above)
Comment (ArthurHeymans) [reply pkg/provisioning/cbnt/tools.go:435]: imageI(nformation). I'll find a better name

File: cmd/cbnt-prov/cmd.go (hunk 1 above)
Comment (ArthurHeymans) [reply cmd/cbnt-prov/cmd.go:419]: This seems to work fine, as coreboot does not specify all arguments. kong does have a flags to set the default value of a command line argument. Should that be used?

Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go:419]: If it works it works

**Review by ChriMarMe — APPROVED** [review]

File: cmd/cbnt-prov/cmd.go (hunk 1 above)
Comment (ArthurHeymans) [reply cmd/cbnt-prov/cmd.go:419]: Are variables not 0 initialized in go? That's probably why it works then for integers.

File: pkg/provisioning/cbnt/tools.go (hunk 2 above)
Comment (ArthurHeymans) [reply pkg/provisioning/cbnt/tools.go]: Done

**Review by ChriMarMe — APPROVED** [review]

**Review by walterchris — APPROVED** [review]

**Pull request merged**